build.gradle

```diff
@@ -91,7 +91,7 @@ sonarqube {
     property "sonar.projectKey", "data-extractor-job"
     property "sonar.coverage.jacoco.xmlReportPaths", "${project.buildDir}/jacoco/test.exec"
     property "sonar.coverage.jacoco.xmlReportPaths", "${project.buildDir}/jacoco/integration.exec"
-
+    property 'sonar.coverage.exclusions', "**/uk/gov/hmcts/reform/dataextractor/**"
```

Comment:
is this erm excluding everything?

Reply:
that is not to merge ... not sure the entire PR is to merge to be honest ... I am testing some alternatives for writing to blob storage and I don't want sonar to complain about their coverage